optaplanner-examples/src/main/java/org/optaplanner/examples/taskassigning_listvariable/domain/solver/TaskDifficultyComparator.java

```diff
@@ -29,6 +29,6 @@
 
     @Override
     public int compare(Task a, Task b) {
-        return COMPARATOR.compare(a, b);
+        return -COMPARATOR.compare(a, b);
```

Comment:
Both difficulty and strength comparators must sort elements in ascending order.

The original `TaskDifficultyComparator` is now used to sort _values_ by _strength_. Stronger means "less difficult", hence the negation that reverses the original ascending difficulty order and makes it an ascending strength order.